src/internal/connector/onedrive/collection.go

```diff
@@ -352,6 +359,21 @@ func (oc *Collection) getDriveItemContent(
 			return nil, clues.Wrap(err, "downloading item").Label(graph.LabelsSkippable)
 		}
 
+		// Skip big OneNote files as they can't be downloaded
+		if clues.HasLabel(err, graph.LabelStatus(http.StatusServiceUnavailable)) &&
+			oc.kind == CollectionKindPackage && *item.GetSize() >= MaxOneNoteFileSize {
+			// FIXME: It is possible that in case of a OneNote file we
+			// will end up just backing up the `onetoc2` file without
+			// the one file which is the important part of the OneNote
+			// "item". This will have to be handled during the
+			// restore, or we have to handle it separately by somehow
+			// deleting the entire collection.
+			logger.CtxErr(ctx, err).With("skipped_reason", fault.SkipBigOneNote).Error("invalid file")
```

Comment:
Nit: skips should be Info level logs (same for the 404 on line 356).  They're known conditions that we're handling.  Not unexpected errors.
```suggestion
			logger.CtxErr(ctx, err).With("skipped_reason", fault.SkipBigOneNote).Info("max size file")
```

Reply:
Ahh, let me make that change in logging to Info. I'm not sure about the "max size file" message though. This file size limit is specific to just OneNote. How about "max OneNote file size exceeded"?